### ISSUE TITLE
Android?

### ISSUE THREAD

**Issue by cookins** [johnboiles/BaofengUV5R-TRRS#14]
hey!
this board will work same for android?

**Comment by cookins** [comment]
Just tried this scheme. My android even can't see headphones plugged in... But PC seeing and even able to send some sounds with VOX...

**Comment by johnboiles** [comment]
I haven’t tested with android but I _think_ it should work! Older android phones had a different TRRS pin out than iPhone but I think they’re the same now

**Comment by Manawyrm** [comment]
Works fine on my nexus 5 and iPhone 8 (with the lightning adapter).
Nice one, thanks for publishing! :)

**Comment by kronicd** [comment]
It works great on my Pixel 4A. It doesn't work reliably on a Xiaomi Mi A1.

@cookins, which software did you use to get it working on the Xiaomi?

**Comment by kronicd** [comment]
@cookins I tested using a 10KΩ resistor on the end of a cable between GND and MIC. The detection works perfectly.

I measured the resistance between GND and MIC on my `[Mi In-Ear Headphones Pro HD](https://www.mi.com/global/headphonesprohd/)` and found they are 8.5KΩ in one direction and 5.5KΩ in the other. I'm hoping that doesn't matter.

I'm going to try replacing R2 and R3 with 8.2KΩ resistors and testing across a variety of mobiles. I don't have an iPhone with a headphone jack though, so if anyone has an old iPhone available to test this change with that would be great.

**Comment by kronicd** [comment]
@johnboiles is there free or free for academic/non-commercial use software you'd recommend for editing the board files?

It'll help me sort things for a pull request later :)

**Comment by johnboiles** [comment]
Yeah it's ~Cadsoft~ Autodesk Eagle! That's free for non commercial use still even after the Autodesk acquisition.

I would love a pull request on this. Thanks for digging into it yall

**Comment by kronicd** [comment]
Adding a 100Ω resistor from LFT to GND appears to have resolved all detection issues.

No other changes were needed.

![Screenshot_20220606-080444](https://user-images.githubusercontent.com/551541/172076176-b45510cf-5a44-40cf-8a36-c6edefe28b57.png)

A through hole component fits well and shouldn't cause any issues with heat shrink. For safety I'd recommend wrapping some kapton tape to insulate against shorts before heat shrinking.

I'll work on a pull request to update the PCB for a more long term solution.

If people could test this in the meantime that would be great.

**Comment by cookins** [comment]
> Adding a 100Ω resistor from LFT to GND appears to have resolved all detection issues.
> 
Our hero. Thanks for work.

**Comment by johnboiles** [comment]
Thanks for figuring this out @kronicd! Maybe about time for a design rev! I've been having great luck getting fully assembled boards from JLCPCB

**Comment by kronicd** [comment]
A [pull request](https://github.com/johnboiles/BaofengUV5R-TRRS/pull/19) has been made to resolve this issue.
